converter-plugins-default/src/main/kotlin/converter/plugins/default/CompressionPlugin.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2025 The Contributors to Eclipse OpenSOVD (see CONTRIBUTORS)
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package converter.plugins.default
+
+import com.google.protobuf.ByteString
+import converter.plugin.api.ChunkApi
+import converter.plugin.api.ConverterApi
+import converter.plugin.api.ConverterPlugin
+import org.apache.commons.compress.compressors.lzma.LZMACompressorOutputStream
+import org.eclipse.opensovd.cda.mdd.Signature
+import java.io.ByteArrayOutputStream
+import java.security.MessageDigest
+import java.util.logging.Logger
+
+/**
+ * Default compression plugin, compresses chunks with LZMA, and adds a sha512 hash of the initial (uncompressed) data.
+ */
+class CompressionPlugin : ConverterPlugin {
+    private val logger = Logger.getLogger("test")
```

Comment:
question: is the loggername "test" on purpose?

Reply:
removed it -- forgot to remove it beforehand when I was testing something, it wasn't used.